### ISSUE TITLE
I should not have to globally install this module

### ISSUE THREAD

**Issue by rclark** [mapbox/secret-shield#9]
Why isn't installing secret-shield as a dependency in my repository sufficient? What's driving the demand that I manage a global npm secret-shield installation?

**Comment by vsmart** [comment]
Hi @rclark, have you seen these instructions to install secret-shield in your repo? If yes what issues have you run into with this? https://github.com/mapbox/secret-shield/blob/master/docs/enabledRepositories.md#how-to-install-in-your-repository

**Comment by rclark** [comment]
Yes, that documentation is incomplete, because when I make a commit, the hook issues this error:

> ERROR! **You must have secret-shield installed and configured globally** to commit to this repository. To set up secret-shield, follow these instructions: https://github.com/mapbox/secret-shield/blob/master/docs/enabledBadge.md

**Comment by rreusser** [comment]
We also ran into problems with projects using [`pre-commit`](https://pre-commit.com/):

```bash
$ pre-commit install
[ERROR] Cowardly refusing to install hooks with `core.hooksPath` set.
hint: `git config --unset-all core.hooksPath`
```

I had to remove secret-shield from my global git hooks.
